test/memcache-helper.js

```diff
@@ -37,6 +36,12 @@ var config = {
   }
 }
 
+const spawn = require('child_process').spawn
```

Comment:
@deeptibaghel I talked to @vbudhram about this and we suspect `spawn` starts the process, but tests timeout because it is not cleaned up after the last test. We could write a function to stop the process, but we can just a simpler solution for now: Start memcached in this file https://github.com/mozilla/fxa-customs-server/blob/master/scripts/test-local.sh and run `kill` on it after tests are done. Edge case to test is the case where it is already running, see what happens.

Example: https://github.com/mozilla/fxa-auth-server/blob/master/scripts/start-local-mysql.sh#L8-L17